node/src/sign.rs

```diff
@@ -115,7 +125,17 @@ pub async fn sign(
         presign_out,
         msg_hash,
     )?;
+    let start = SystemTime::now();
+
     let signature = run_protocol("sign", channel, me, protocol).await?;
+
+    let duration = start.elapsed()?.as_millis();
```

Comment:
There is a RAII guard provided by HistogramTimer (`.start_timer()` on a Histogram, see nearcore repo for examples) that will do this for you.

It will also ensure that in case run_protocol returns an error, the elapsed time will still be counted.

btw, for timing purposes, use Instant instead of SystemTime; Instant is guaranteed to be monotonic.